src/matching/Generic_vs_generic.ml

```diff
@@ -2237,15 +2237,14 @@ and m_stmts_deep ~inside ~less_is_ok (xsa : G.stmt list) (xsb : G.stmt list) =
       m_stmts_deep ~inside ~less_is_ok xsa bbs
   | ( ({ s = G.ExprStmt ({ e = G.Ellipsis _i; _ }, _); _ } :: _ as xsa),
       (_ :: _ as xsb) ) ->
-      (* let's first try without going deep *)
-      m_list__m_stmt xsa xsb >!> fun () ->
```

Comment:
each time we use >!>, there is a bug because we cut some matches.
Ideally we should remove this combinator.
Ideally we should use CTL instead of those deep stmt hacks.